README.md

```diff
@@ -1 +1,17 @@
-# python-swmrtools
+# swmr_tools
+
+swmr_tools is a Python library developed by Diamond Light Source ltd designed to facilitate reading and processing data from nexus (hdf5) files as they are being written
+
+## Installation
+
+Use the package manager [pip](https://pip.pypa.io/en/stable/) to install swmr_tools
+
+'''bash
+pip install swmr_tools
+'''
+
+## Usage
+
+- For full examples of use please see the full documentation at https://iterator.readthedocs.io/en/latest/
```

Comment:
Should be a link to new rtd site

Reply:
Linked to what I think the new site will be, once the pull request has gone through I will create a new readthedocs site, point it to this repo and check that the url matches what I think it will.